sample_projects/LEQuad/lequad.cpp

```diff
@@ -0,0 +1,629 @@
+    ret &= mavlink_communication.onboard_parameters().add_parameter_float(&position_estimation.kp_pos_gps[0], "POS_KP_POS0"     );
+    ret &= mavlink_communication.onboard_parameters().add_parameter_float(&position_estimation.kp_pos_gps[1], "POS_KP_POS1"     );
+    ret &= mavlink_communication.onboard_parameters().add_parameter_float(&position_estimation.kp_pos_gps[2], "POS_KP_POS2"     );
+    ret &= mavlink_communication.onboard_parameters().add_parameter_float(&position_estimation.kp_vel_gps[0], "POS_KP_VEL0"     );
+    ret &= mavlink_communication.onboard_parameters().add_parameter_float(&position_estimation.kp_vel_gps[1], "POS_KP_VEL1"     );
+    ret &= mavlink_communication.onboard_parameters().add_parameter_float(&position_estimation.kp_vel_gps[2], "POS_KP_VEL2"     );
+
+    // Data logging
+    ret &= data_logging_continuous.add_field(&position_estimation.local_position.pos[0], "local_x", 3);
+    ret &= data_logging_continuous.add_field(&position_estimation.local_position.pos[1], "local_y", 3);
+    ret &= data_logging_continuous.add_field(&position_estimation.local_position.pos[2], "local_z", 3);
+    ret &= data_logging_stat.add_field(&position_estimation.local_position.origin.latitude,  "origin_lat", 7);
+    ret &= data_logging_stat.add_field(&position_estimation.local_position.origin.longitude, "origin_lon", 7);
+    ret &= data_logging_stat.add_field(&position_estimation.local_position.origin.altitude,  "origin_alt", 3);
+
+    return ret;
+}
+
+// -------------------------------------------------------------------------
+// Stabilisers
+// -------------------------------------------------------------------------
+bool LEQuad::init_stabilisers(void)
+{
+    bool ret = true;
+
+    // -------------------------------------------------------------------------
+    // Stabilisation copter
+    // -------------------------------------------------------------------------
+    // Module
+    ret &= stabilisation_copter_init(&stabilisation_copter,
+                                    config_.stabilisation_copter_config,
+                                    &controls,
+                                    &ahrs,
+                                    &position_estimation,
+                                    &command.torque,
+                                    &command.thrust);
+    ret &= stabilisation_init(&controls);
+
+    // Parameters
+    Onboard_parameters& op            = mavlink_communication.onboard_parameters();
+    stabiliser_t* rate_stabiliser     = &stabilisation_copter.stabiliser_stack.rate_stabiliser;
+    stabiliser_t* attitude_stabiliser = &stabilisation_copter.stabiliser_stack.attitude_stabiliser;
+    stabiliser_t* velocity_stabiliser = &stabilisation_copter.stabiliser_stack.velocity_stabiliser;
+    ret &= op.add_parameter_float(&rate_stabiliser->rpy_controller[ROLL].p_gain,                    "ROLL_R_KP");
```

Comment:
You mean have a function `add_parameter` that does not add a parameter ? :-1:

This is a case where you would override the function.